src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -3,35 +3,44 @@ import {
   DebugElement,
   Input
 } from '@angular/core';
+
```

Comment:
The addition of the new lines between imports effects the linter. The imports are grouped alphabetically by the library name, not the imported component. The linter only looks at imports that have no new lines. 

The layout that I tried to use was Angular in the first group, then external, then internal. On very large imports I might break it down further.

On a different note, if you don't want the linter to check the order of these we can have that discussion. But I see no value in the new lines.